spec/unit/upgrade_format_task_spec.rb

```diff
@@ -87,22 +85,28 @@ def migrate; end
       end
     end
 
+    it "lists each system description and its status during upgrade" do
+      expected_output = <<-EOF
+Reading "description1" ... successfully upgraded from version 1 to #{SystemDescription::CURRENT_FORMAT_VERSION}.
+Reading "description2" ... No upgrade necessary.
```

Comment:
There is some inconsistency in capitalization. In the first part of the message it says "... successfully" in the second part "... No". It should either both be upper case or both be lower case. I think I would prefer upper case, as it actually is two sentences.